taxi_service/settings.py

```diff
@@ -122,7 +122,11 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
+
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
```

Comment:
The STATICFILES_DIRS setting is a list of filesystem directories to check when loading static files. It should contain the full path to your static directory. You should use BASE_DIR / 'static' instead of just BASE_DIR.